Formula/libbi.rb

```diff
@@ -0,0 +1,156 @@
+class Libbi < Formula
+  desc "Bayesian state-space modelling on parallel computer hardware"
+  homepage "http://libbi.org"
+  head "https://github.com/libbi/LibBi.git"
+
+  stable do
+    url "https://github.com/libbi/LibBi/archive/1.3.0.tar.gz"
+    sha256 "0dd313dd71e72b2f16ca9074800fc2fa8bf585bec3b87a750ff27e467a9826d0"
+  end
+
+  option "without-test", "Disable build-time checking (not recommended)"
+
+  depends_on "perl"
+  depends_on "qrupdate"
+  depends_on "netcdf"
+  depends_on "gsl"
+  depends_on "boost"
+  depends_on "automake" => :run
```

Comment:
Are you sure automake is needed at run-time? (and not autoconf or libtool?) This is unusual

Reply:
automake and autoconf are both required because C++ code is generated and compiled at run-time. Should autoconf be put here explicitly even though automake already depends on it?

Reply:
No that should be okay then